templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
Your comment below is the first I've seen about api-external-dns-names... I think this can actually work:

If we create a LB resource in CFn we should be able to use `Fn::GetAtt` to grab its DNSName (see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html), and then just pass that value to the startup script as a api-external-dns-names flag to `kubeadm init`.  We obviously wouldn't know what backend ports to configure but I think the cloud provider should handle it so long as we tag it with the right KubernetesCluster.

I'll try this and see if it can work better.